src/Compilers/Test/Utilities/CSharp/BasicCompilationUtils.cs

```diff
@@ -37,7 +37,9 @@ private static VisualBasicCompilation CreateCompilationWithMscorlib(string sourc
             return VisualBasicCompilation.Create(assemblyName, new[] { tree }, references, options);
         }
 
-        private static BasicTestBase s_instance = new BasicTestBase();
+        private static BasicTestBase s_instance;
```

Comment:
What is the reason behind this change? Are we trying to get rid of static constructors? Why?